.github/workflows/test_weaver-data-sharing.yaml

```diff
@@ -525,13 +525,13 @@ jobs:
       # CORDA DRIVER
       - name: Start Corda Driver
         run: |
-          sed -i "s#^DOCKER_IMAGE_NAME=.*#DOCKER_IMAGE_NAME=weaver-corda-driver#g" docker-testnet-envs/.env.corda
+          sed -i "s#^DOCKER_IMAGE_NAME=.*#DOCKER_IMAGE_NAME=cacti-weaver-corda-driver#g" docker-testnet-envs/.env.corda
```

Comment:
Now that we are publishing these packages afresh in a new repository, I think we ought to name them in a consistent way, with some sense of hierarchy. So I'd suggest naming this to `cacti-weaver-driver-corda`.